lib/Doctrine/DBAL/Schema/Sequence.php

```diff
@@ -22,74 +22,44 @@ class Sequence extends AbstractAsset
     /** @var int|null */
     protected $cache = null;
 
-    /**
-     * @param string   $name
-     * @param int      $allocationSize
-     * @param int      $initialValue
-     * @param int|null $cache
-     */
-    public function __construct($name, $allocationSize = 1, $initialValue = 1, $cache = null)
+    public function __construct(string $name, int $allocationSize = 1, int $initialValue = 1, ?int $cache = null)
     {
         $this->_setName($name);
         $this->setAllocationSize($allocationSize);
         $this->setInitialValue($initialValue);
         $this->cache = $cache;
     }
 
-    /**
-     * @return int
-     */
-    public function getAllocationSize()
+    public function getAllocationSize() : int
     {
         return $this->allocationSize;
     }
 
-    /**
-     * @return int
-     */
-    public function getInitialValue()
+    public function getInitialValue() : int
     {
         return $this->initialValue;
     }
 
-    /**
-     * @return int|null
-     */
-    public function getCache()
+    public function getCache() : ?int
     {
         return $this->cache;
     }
 
-    /**
-     * @param int $allocationSize
-     *
-     * @return \Doctrine\DBAL\Schema\Sequence
-     */
-    public function setAllocationSize($allocationSize)
+    public function setAllocationSize(int $allocationSize) : self
     {
-        $this->allocationSize = (int) $allocationSize ?: 1;
+        $this->allocationSize = $allocationSize ?: 1;
```

Comment:
Is the default value still relevant? In the past, it might cover some NULL value which is no longer relevant.